neqo-transport/src/cc.rs

```diff
@@ -243,6 +254,7 @@ impl CongestionControl {
         if !self.in_congestion_recovery(sent_time, PacketState::Lost) {
             self.congestion_recovery_start_time = Some(now);
             self.congestion_window /= 2; // kLossReductionFactor = 0.5
+            self.acked_bytes /= 2;
```

Comment:
I think this should be set to congestion_window/2 (i.e. the new cwnd)

Reply:
I don't think so.  acked_bytes is the number of bytes that we haven't deposited into the congestion window.  

In slow start, this value will always be zero (because we always transfer from acked_bytes to congestion_window), so setting a non-zero value will result in an increase to the congestion window.

In congestion avoidance, this value will be between 0 and the current congestion window (that is, once it goes above the congestion window, we drop the value down and add MAX_DATAGRAM_SIZE to the congestion window).

This is just scaling the `0..congestion_window` value to `0..(congestion_window/2)` to match the new value for congestion_window.

Reply:
Thinking about it a bit this should be sett to 0. we should start counting from 0 again.

Reply:
Also need to set acked_bytes = 0 when persistent congestion event.